src/Bitmovin/api/factories/muxing/MuxingFactory.php

```diff
@@ -21,15 +22,26 @@
 use Bitmovin\api\model\outputs\Output;
 use Bitmovin\configs\manifest\DashOutputFormat;
 use Bitmovin\configs\manifest\HlsOutputFormat;
+use Bitmovin\configs\manifest\ProgressiveMp4OutputFormat;
 use Bitmovin\configs\manifest\SmoothStreamingOutputFormat;
 
 class MuxingFactory
 {
 
+    /**
+     * @param Encoding  $encoding
+     * @param Stream    $stream
+     * @param           $output
+     * @param           $outputPath
+     * @param ApiClient $apiClient
+     *
+     * @return FMP4Muxing
+     * @throws \Bitmovin\api\exceptions\BitmovinException
+     */
     private static function createFMP4Muxing(Encoding $encoding, Stream $stream, $output, $outputPath, ApiClient $apiClient)
     {
-        $encodingOutput = null;
-        if ($output != null && $outputPath != null)
+        $encodingOutput = NULL;
```

Comment:
As it basically a constant, I tend to write null, true, false in uppercase. However, reverted it